src/main/java/appeng/container/implementations/ContainerCraftAmount.java

```diff
@@ -38,9 +40,9 @@
 public class ContainerCraftAmount extends AEBaseContainer
 {
 
-	public final Slot craftingItem;
-	final ITerminalHost priHost;
-	public IAEItemStack whatToMake;
+	private final Slot craftingItem;
+	private final ITerminalHost priHost;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "priHost" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)